src/frontend/ts/state/chessState.ts

```diff
@@ -1,22 +1,26 @@
 import Urbit from '@urbit/http-api'
-import { Ship, GameID, SAN, FENPosition, Move, GameInfo, ActiveGameInfo, Challenge, ChessUpdate, ChallengeUpdate } from '../types/urbitChess'
+import { Ship, GameID, SAN, FENPosition, Move, GameInfo, ActiveGameInfo, ArchiveGameInfo, Challenge, ChessUpdate, ChallengeUpdate } from '../types/urbitChess'
 
 interface ChessState {
   // properties
   urbit: Urbit | null
-  displayGame: ActiveGameInfo | null
+  displayGame: ActiveGameInfo | ArchiveGameInfo | null
   displayIndex: number | null
   practiceBoard: String | null
   activeGames: Map<GameID, ActiveGameInfo>
+  localArchive: Map<GameID, ArchiveGameInfo>
```

Comment:
for this one, I agree that it sounds confused with `localStorage`. but I wanted to invoke the idea that the archived games in state is an incomplete list from the `archive` on urbit.

and now that I'm working on the _archive searching scries_ it might make sense to store scry results in `localStorage` so that we don't wipe search results from users. 
right now, every time a user clicks on **Archive** it runs a scry and repopulates the list. the same thing happens when the front end receives a _result update_.

Reply:
If you look in `App.tsx` you can see that I have it scrying the `friends` wire as soon as the app is opened, so that the Friends menu is populated before you click on it.